python/ray/tune/integration/docker.py

```diff
@@ -35,6 +39,9 @@ def __init__(self,
                  local_dir: str,
                  remote_dir: str,
                  sync_client: Optional[SyncClient] = None):
+        configure_logging(
+            log_style="record",
+            verbosity=env_integer("TUNE_SYNCER_VERBOSITY", 0))
```

Comment:
Does this change logging behavior just for the syncer or for all Tune output?

Reply:
just syncer